requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest-cov
```

Comment:
Do we have a real need in these packages except from the CI build stage?
I notice that these dependencies have been already added in `main.yml`.